.github/workflows/phpunit.yml

```diff
@@ -57,3 +58,6 @@ jobs:
 
       - name: "Tests"
         run: "vendor/bin/phpunit"
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
```

Comment:
Do we need this token if it's a public repo?

Reply:
Yes, it Is required to configure the Codecov badge